### PR TITLE
support Dump external call relations

### PR DESCRIPTION
**Pull request by ByteSecurity**
- change args corpus_path to work_dir
- write call relations to work_dir/relations.log

### CODE REVIEW

**Comment by ByteSecurity** [comment]
add oracles:typed_debug(x)

File: src/evm/host.rs [hunk 1]
```diff
@@ -362,6 +378,40 @@ where
...
+        if self.relations_hash.contains(&cur_wirte_hash) {
+            return;
+        }
+        if self.relations_hash.len() == 0{
+            let write_head = format!("[ityfuzz relations] caller, traget, function hash\n");
+            self.relations_file
+                .write_all(write_head.as_bytes())
+                .unwrap();
+        }else {
+            self.relations_file
```
Comment (shouc) [inline src/evm/host.rs:398]: 不应该是 caller --> target吗，你这是(caller1, target1) -->  (caller2, target2)吧

**Review by shouc — CHANGES_REQUESTED** [review]

File: src/evm/host.rs (hunk 1 above)
Comment (ByteSecurity) [reply src/evm/host.rs:398]: 好嘞，已修复

**Comment by shouc** [comment]
closed in favor of https://github.com/fuzzland/ityfuzz/pull/89

**Pull request closed without merging**